fms/utils/serialization.py

```diff
@@ -114,6 +114,51 @@ def _legacy_attn_unfused_to_fused_adapter(orig_sd):
     return new_sd
 
 
+def _legacy_mlp_glu_unfused_to_fused_adapter(orig_sd):
+    """
+    Legacy adapter for converting pre 0.0.6 unfused mlp glu weights to fused mlp glu weights
+    """
+    new_sd = {}
+    removed_params = set()
+    orig_keys = set(orig_sd.keys())
+    for name in orig_keys:
+        # this keys been popped and we dont need to process it
+        if name in removed_params:
+            continue
+
+        if (
+            "ff_sub_layer.wg_fused" not in name
+            and "ff_sub_layer.wg" in name
+            or "ff_sub_layer.w1" in name
```

Comment:
this should probably have parens, `x and y or z` is a bit confusing from a precedence perspective

Reply:
fixed in latest commit